features/upgrade/sdn/externalIP-upgrade.feature

```diff
@@ -60,12 +60,12 @@ Feature: SDN externalIP compoment upgrade testing
   @proxy @noproxy @disconnected @connected
   Scenario: Check the externalIP works well after upgrade
     Given I switch to cluster admin pseudo user
-    Given I store the schedulable nodes in the :nodes clipboard
-    And the Internal IP of node "<%= cb.nodes[0].name %>" is stored in the :hostip clipboard
+    # Get the external ip from  service
     When I use the "externalip-upgrade" project
+    And evaluation of `service('service-unsecure').raw_resource(cached: true).dig('spec','externalIPs')[0]` is stored in the :hostip clipboard
     Given a pod becomes ready with labels:
       | name=externalip-pod    |
-    And evaluation of `pod(1).name` is stored in the :pod1name clipboard
+    And evaluation of `pod(0).name` is stored in the :pod1name clipboard
```

Comment:
Simply use `pod.name` should work here.

Reply:
Thanks ,fixed it.

Reply:
Latest passed log:
https://privatebin.corp.redhat.com/?75d69b44b3a477fc#4XNGNcfWKFJ5rpcVQze635Mdd81H8JCGNvnYyfkXVZkw